Terran-One/cw-simulate-ui
# Slim Menu

## Description
Remove hierarchy menu in favor of drawer bar icons & slim submenus. Submenu header color is placeholder.

Currently removed delete options due to incompatibility with new `CWSimulateApp`. Also factored out `Options` component for reuse.

## Test steps
1. Create new simulation.
2. Drawer bar should have 5 icons w/ tooltips
3. Contracts icon should have a submenu w/ header & uploaded contracts. Each contract has an Options menu w/ Instantiate item.
4. Instances icon should have a submenu w/ header & contract instances. Each instance has an Options menu w/ Copy Address item.
5. Remaining icons lead to other pages of the app
